### PR TITLE
Adding changleog and using grunt release to update

### PR DESCRIPTION
**Pull request by jodiedoubleday**
#### What does this PR do?
- Adds a CHANGELOG.md
- Adds auto update to Grunt Release for CHANGELOG.md

#### What unit or integration tests does this PR have?
None

#### What selenium tests does this PR have?
None

#### How should a developer review this?
- run `grunt release --no-write` this should fake a release.
- The status of running this command should show you that it has changed and deployed the changelog (though the result will be blank)

#### How should this be manually tested?
As above

#### Any background context you want to provide?
This repo didn't have a changelog

#### What are the relevant tickets?
https://holidayextras.jira.com/browse/UXUI-237

#### Screenshots (if appropriate)
Command Line

#### What gif best describes how you feel about this work?
All change please
---
#### Developer Definition of Done/Quality Checklist (for PR author to complete BEFORE code review):
- [x] I've updated documentation with any changes to procedures.
- [x] I've checked this work against the requirements of the Jira.
- [x] I've checked that there are no negative speed or performance impacts from my work.
- [x] I am ready for this to be code reviewed, merged and tested.
- [x] I've thought about writing a blog post about this.

#### IP or Developer Review:
- [x] I agree with the assumptions made in the quality checklist.
- [x] I’ve witnessed the work behaving as expected.
- [x] +1 from me!

#### Software Engineer or Developer review:
- [x] I agree with the assumptions made in the quality checklist.
- [x] I’ve witnessed the work behaving as expected.
- [x] I’ve checked for appropriate test coverage.
- [x] I’ve checked for coding anti-patterns.
- [x] I've checked this work against the requirements of the Jira.
- [x] +1 from me!

#### Software Engineer or project guru review:
- [x] I agree with the assumptions made in the quality checklist.
- [x] I’ve witnessed the work behaving as expected.
- [ ] I’ve run all the tests locally and they pass.
- [x] I’ve checked for coding anti-patterns.
- [x] I don’t believe this work introduces unnecessary technical debt.
- [x] +1 from me!

### CODE REVIEW

**Comment by HXJackObeney** [comment]
Taking IP review

**Comment by HXJackObeney** [comment]
Looks good +1

**Comment by dancork** [comment]
Taking dev

File: grunt/release.js [hunk 1]
```diff
@@ -1,5 +1,7 @@
 module.exports = {
   options: {
+    changelog: true,
+    changelogText: '<%= version %>\n',
```
Comment (jackdcrawford) [inline grunt/release.js]: I'd say the default `changelogText` is better than this: https://github.com/geddski/grunt-release#options

**Comment by dancork** [comment]
The dry run appears to work fine :+1:

**Comment by HeinBehrenshx** [comment]
+1

**Pull request merged**